src/autoComplete/index.tsx

```diff
@@ -248,7 +278,7 @@ export default class AutoComplete extends Component<AutoCompleteProps, AutoCompl
           onBlur={this.props.onBlur}
           onClear={this.props.onClear}
           placeholder={this.props.placeholder}
-          defaultValue={String(this.state.query)}
+          defaultValue={String(this.state.textfieldValue)}
```

Comment:
this.state.textfieldValue is already a string

Reply:
Done